UnityProject/Assets/Scripts/Tilemaps/Tiles/ObjectTile.cs

```diff
@@ -23,48 +23,48 @@ public class ObjectTile : LayerTile
 		public WiringColor CableType;
 		public Sprite wireSprite;
 
-#if UNITY_EDITOR
-		private void OnValidate()
-		{
-			if (Object != null)
-			{
-				if (objectCurrent == null)
-				{
-					// if sprite already exists (e.g. at startup), then load it, otherwise create a new one
-					EditorApplication.delayCall += () =>
-					{
-						if(IsWire && wireSprite != null)
-						{
-							PreviewSprite = wireSprite;
-						}
-						else
-						{
-						PreviewSprite = PreviewSpriteBuilder.LoadSprite(Object) ??
-						                PreviewSpriteBuilder.Create(Object);
-						}
-					};
-				}
-				else if (Object != objectCurrent)
-				{
-					// from one object -> other (overwrite current sprite)
-					EditorApplication.delayCall += () => { PreviewSprite = PreviewSpriteBuilder.Create(Object); };
-				}
-			}
-			else if (objectCurrent != null)
-			{
-				// setting to None object (delete current sprite)
-				GameObject obj = objectCurrent;
-				EditorApplication.delayCall += () => { PreviewSpriteBuilder.DeleteSprite(obj); };
-			}
-
-			objectCurrent = Object;
-
-			if (objectCurrent != null && objectCurrent.GetComponentInChildren<RegisterItem>() != null)
-			{
-				IsItem = true;
-			}
-		}
-#endif
+//#if UNITY_EDITOR
```

Comment:
Can we just remove this whole comment block all together?

Reply:
Not too sure what to do about it because if you were to get stuck infinitely generating textures, but I don't know what it's exactly used for

Reply:
alright leave it for the time being